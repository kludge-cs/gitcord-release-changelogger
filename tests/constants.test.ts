import consts from "../src/constants";

describe("constants (STATIC)", () => {
	test("SHOULD match snapshot", () => {
		expect(consts).toMatchSnapshot("constants");
	});
});
