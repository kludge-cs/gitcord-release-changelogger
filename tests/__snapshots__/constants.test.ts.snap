// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`constants (STATIC) SHOULD match snapshot: constants 1`] = `
Object {
  "BODY_MISSING": "# Meta
This release has no bundled changelog.",
}
`;
